Areas/Identity/IdentityHostingStartup.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using RPauth.Data;
+using whale_spotting.Models.Database;
+
+
+[assembly: HostingStartup(typeof(whale_spotting.Areas.Identity.IdentityHostingStartup))]
+namespace whale_spotting.Areas.Identity
+{
+    public class IdentityHostingStartup : IHostingStartup
+    {
+        public void Configure(IWebHostBuilder builder)
+        {
+            builder.ConfigureServices((context, services) => {
+                services.AddDbContext<WhaleSpottingContext>(options =>
+                    // options.UseSqlServer(
```

Comment:
can we delete the comment and also change this to use the ConnectionHelper I've created now like in WhaleSpottingContext.cs please :)